src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -1010,3 +1010,103 @@ TEST_F(SensorFusionTest, breakbeam_fail_test_ssl)
     // did it not use robot position
     EXPECT_TRUE(ball_position != robot_state.position());
 }
+
+TEST_F(SensorFusionTest, breakbeam_in_robot_test)
+{
+    // Check that breakbeam status is propagated to the Robot
+    SensorProto sensor_msg;
+
+    Point ball_position_ssl(0.75, 0.75);
+    Vector velocity(0, 0);
+    ball_state = BallState{ball_position_ssl, velocity};
+
+    yellow_robot_states.clear();
+    blue_robot_states.clear();
+
+    RobotState robot_state(Point(0, 0), Vector(0, 0), Angle::fromRadians(2),
+                           AngularVelocity::zero());
+    RobotStateWithId robot_id = {2, robot_state};
+    yellow_robot_states.push_back(robot_id);
+
```

Comment:
these variables seem unused

Reply:
I believe 1019 to 1029 can be deleted. see https://github.com/UBC-Thunderbots/Software/pull/3462/files#r2136549546